.github/workflows/build.yml

```diff
@@ -201,6 +212,8 @@ jobs:
         env:
           JAVA_HOME: /home/gitpod/.sdkman/candidates/java/current
           VERSION: ${{needs.configuration.outputs.version}}
+          LEEWAY_REMOTE_CACHE_BUCKET: ${{needs.configuration.outputs.leeway_cache_bucket}}
+          IMAGE_REPO_BASE: ${{needs.configuration.outputs.image_repo_base}}/dev
```

Comment:
How come this uses /dev sub-folder for build? 

Should this step not run if `github.ref == 'refs/heads/main'`?